src/GraphQL/Types/DirectiveGraphType.cs

```diff
@@ -73,14 +73,30 @@ public enum DirectiveLocation
 
     /// <summary>
     /// Directives are used by the GraphQL runtime as a way of modifying execution
-    /// behavior.Type system creators will usually not create these directly.
+    /// behavior. Type system creators will usually not create these directly.
     /// </summary>
     public class DirectiveGraphType : INamedType
     {
+        /// <summary>
+        /// Returns a static instance of the predefined 'include' directive.
+        /// </summary>
         public static readonly IncludeDirective Include = new IncludeDirective();
+
+        /// <summary>
+        /// Returns a static instance of the predefined 'skip' directive.
+        /// </summary>
         public static readonly SkipDirective Skip = new SkipDirective();
+
+        /// <summary>
+        /// Returns a static instance of the predefined 'deprecated' directive.
+        /// </summary>
         public static readonly GraphQLDeprecatedDirective Deprecated = new GraphQLDeprecatedDirective();
 
+        /// <summary>
+        /// Initializes a new instance with the specified parameters.
+        /// </summary>
+        /// <param name="name">The type name within the GraphQL schema.</param>
```

Comment:
```suggestion
        /// <param name="name">The directive name within the GraphQL schema.</param>
```

It is important - the directive is not a graph type. We've got issues with this in some places.

Reply:
By the way, the question is about quality of `INamedType` interface name :)